plugins/optimization-detective/storage/rest-api.php

```diff
@@ -131,6 +131,7 @@ function od_handle_rest_request( WP_REST_Request $request ) {
 		$data = $request->get_params();
 		// Remove params which are only used for the REST API request and which are not part of a URL Metric.
 		unset(
+			$data['_wpnonce'],
 			$data['slug'],
 			$data['nonce']
 		);
```

Comment:
This may be short-sighted. See https://github.com/WordPress/performance/issues/1567

Reply:
I originally did this in 1da219f394d546802d8b7b2c43f936152611cd59 (and 5db6f54d9d1196501008fdbcfc7206b99b75e7b0) but then reverted it in 0fa263a75ef90c0f404c71621b1d7cb9dd5482a1 should be reverted. I need to remember why I undid that.

Reply:
See https://github.com/WordPress/performance/pull/1574